### DISCUSSION TITLE
Arming of a swarm

### DISCUSSION THREAD

**Discussion by matejkarasek** [bitcraze/discussions#1358] [Q&A] [answered]
Hi, I would like to fly a swarm of crazyflies that require arming (CONFIG_MOTORS_REQUIRE_ARMING=y) like in the [synchornizedSequence.py](https://github.com/bitcraze/crazyflie-lib-python/blob/master/examples/swarm/synchronizedSequence.py) example:

In older firmware, you could just arm/disarm the crazyflie by setting a parameter to true/false, which was easy to do with swarm.parallel_safe(...).

Now that the arming is handled via the [CRTP arming message](https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/functional-areas/crtp/crtp_platform/#armdisarm-system), I didn't manage to find a solution yet.

Thanks for any advice!

**Comment by matejkarasek** [comment]
I expect I need to use the [send_arming_request](https://github.com/bitcraze/crazyflie-lib-python/blob/c7eaabe31612e796fcdcf8511c18d197051e17cd/cflib/crazyflie/platformservice.py#L91) function, but did not figure out how to do that for a swarm...

**Reply by knmcguire** [reply]
So.. adding `cf.platformservice.send_arming_request(True) `in the run_sequence doesn't work for you?

**Reply by matejkarasek** [reply, marked as answer]
Ah, yeah, that was a pointer in the right direction...

The platformservice, where the [send_arming_request](https://github.com/bitcraze/crazyflie-lib-python/blob/c7eaabe31612e796fcdcf8511c18d197051e17cd/cflib/crazyflie/platformservice.py#L91) function sits, is initialized as "platform", so adding `cf.platform.send_arming_request(True)` did the trick!

Thanks! (and apologies for the late reply, only got to try this now...)
